### PR TITLE
Opal 920 use ghcr

### PR DESCRIPTION
**Pull request by IM-USAF**

### CODE REVIEW

File: .github/workflows/conda-build.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ on:
     branches:
       - master
 jobs:
+  call-ci-image-update-job:
+    uses: ./.github/workflows/ubuntu-ci-image-update.yml
```
Comment (RR-USAF) [inline .github/workflows/conda-build.yml:11]: supurfluous, this gets called as part of call-ubuntu-build-job

File: .github/workflows/ubuntu-ci-image-update.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+name: ubuntu-ci-image-build
+on:
+  workflow_call:
+env: 
+  DOCKERFILE_PATH: ${{ vars.CI_UBUNTU_DOCKERFILE_PATH }}
+  CI_IMAGE_NAME: ${{ vars.CI_UBUNTU_IMAGE_NAME }}
+  CI_IMAGE_TAG: ${{ vars.CI_UBUNTU_IMAGE_TAG }}
+jobs:
+  check-changed-dockerfile-job:
+    runs-on: ubuntu-latest
```
Comment (RR-USAF) [inline .github/workflows/ubuntu-ci-image-update.yml:10]: consider pinning ubuntu version (22.04)?

**Review by RR-USAF — APPROVED** [review]

File: .github/workflows/conda-build.yml (hunk 1 above)
Comment (IM-USAF) [reply .github/workflows/conda-build.yml:11]: removed

File: .github/workflows/ubuntu-ci-image-update.yml (hunk 2 above)
Comment (IM-USAF) [reply .github/workflows/ubuntu-ci-image-update.yml:10]: fixed

**Pull request merged**